enterprise/server/backends/memcache/memcache.go

```diff
@@ -34,6 +41,23 @@ type Cache struct {
 	prefix string
 }
 
+func Register(env *real_environment.RealEnv) error {
+	if fmt.Sprintf("%T", env.GetCache()) == fmt.Sprintf("%T", composable_cache.NewComposableCache(nil, nil, 0)) {
```

Comment:
nit: maybe we should export composableCache.ComposableCache so that we can do these checks without resorting to stringification.

Then we can just write:

if _, ok := env.GetCache().(*composable_cache.ComposableCache); ok {
  // warning time
}

Reply:
Done.